src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -52,11 +60,20 @@ const defaultProps = {
     onSelectRow: () => {},
     isFocused: false,
     style: null,
+    comment: '',
 };
 
 function OptionRowLHN(props) {
     const optionItem = SidebarUtils.getOptionData(props.reportID);
 
+    useEffect(() => {
+        if (!optionItem) return;
+        if (!optionItem.hasDraftComment && props.comment && props.comment.length > 0 && !props.isFocused) {
+            Report.setReportWithDraft(props.reportID, true);
+        }
```

Comment:
@mkarkachov Can we move `!optionItem.hasDraftComment && props.comment && props.comment.length > 0 && !props.isFocused` the logic to early return condition?